docs/uninstall_and_downgrade.md

```diff
@@ -0,0 +1,244 @@
+# SAT Uninstall and Downgrade
+
+## Uninstall: Remove a Version of SAT
+
+This procedure can be used to uninstall a version of SAT.
+
+### Prerequisites
+
+- Only versions 2.2 or newer of SAT can be uninstalled with `prodmgr`. Older versions must be uninstalled manually.
+- CSM version 1.2 or newer must be installed, so that the `prodmgr` command is available.
+
+### Procedure
+
+1. Use `sat showrev` to list versions of SAT.
+
+   **Note:** It is not recommended to uninstall a version designated as "active".
+   If the active version is uninstalled, then the activate procedure must be executed
+   on a remaining version.
+
+   ```screen
+   ncn-m001# sat showrev --products --filter product_name=sat
+   ###############################################################################
+   Product Revision Information
+   ###############################################################################
+   +--------------+-----------------+--------+-------------------+-----------------------+
+   | product_name | product_version | active | images            | image_recipes         |
```

Comment:
We should remove the `active` column here and consider how to revise the Note above which mentions it.

It's probably better done under a separate Jira, and you might already have one.

Reply:
I created [STP-3520](https://jira-pro.its.hpecorp.net:8443/browse/STP-3520) to update this column/wording throughout the documentation in a future PR.